src/js/fc.js

```diff
@@ -501,5 +501,16 @@ var FC = {
 
     getMcuType: function () {
         return FC.MCU_TYPES[CONFIG.mcuTypeId];
-    }
+    },
+
+    boardHasVcp() {
```

Comment:
The definition of other functions in this file has the format `name: function () { }` I think is good if we maintain the same style for all of them.

Reply:
Agreed. This was copy / pasta.